lib/index.js

```diff
@@ -162,7 +191,9 @@ class Linter {
     }
 
     let rules = this.buildRules({
```

Comment:
I would prefer `buildRules` do the logic to decide the final rule configuration to use (instead of passing in `overrideRules`)